fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirSchemaVersion.java

```diff
@@ -28,6 +28,9 @@
     ,V0011(11, "issue-2011 add LAST_UPDATED to each xxx_LOGICAL_RESOURCES table")
     ,V0012(12, "issue-2109 add VERSION_ID to each xxx_LOGICAL_RESOURCES table")
     ,V0013(13, "Add $erase operation for hard delete scenarios")
+    ,V0014(14, "whole-system search and canonical references")
+    ,V0015(15, "issue-2155 add parameter hash to bypass update during reindex")
+    ,V0016(16, "issue-1921 add dedicated common_token_values mapping table for security")
```

Comment:
consider changing "for security" to "for _security" or "for searching on security labels" to make it clear that this is for the search feature and not about the security of the db itself

Reply:
good one